go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/doresoap/ytools
+
+go 1.14
+
+require (
+	github.com/codesoap/ytools v0.0.0-20191116080901-b89db6bddc91 // indirect
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
+)
```

Comment:
This doesn't look quite right. `github.com/codesoap/ytools` should not be a dependency, since it is included in this module. `golang.org/x/net` is a direct dependency and shouldn't be marked as indirect. I think this may be a result of the typo above.

Reply:
Correct.